src/Resources/config/rc.xml

```diff
@@ -5,7 +5,7 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="zicht_admin.controller.rc" class="Zicht\Bundle\AdminBundle\Controller\RcController">
+        <service id="zicht_admin.controller.rc" class="Zicht\Bundle\AdminBundle\Controller\RcController" public="true">
```

Comment:
Misschien meteen nieuwe style naamgeving toepassen, met alias voor backwards compatability?

Reply:
Mja ik heb ongeveer 15 pr's open staan in diverse bundles + de hoofd-PR in TTC, er komen nog wel meer van dit soort kleine wijzigingen langs. 

Je zou namelijk ook de `public=true` eigenlijk niet willen toevoegen maar via `DI` willen afdwingen welke services je gebruikt. Maar dat is niet altijd even makkelijk.

We gaan wsl ook straks de TTV-omgeving updaten, ik stel voor dat we daar wat tijd steken in deze verbeteringen, met een mooi BC plan en een notificatie dat in een volgende major release dit er uit gaat bijv.